stock_available_to_promise_release/models/stock_move.py

```diff
@@ -49,9 +49,51 @@ class StockMove(models.Model):
         search="_search_release_ready",
     )
     need_release = fields.Boolean(index=True, copy=False)
+    unrelease_allowed = fields.Boolean(compute="_compute_unrelease_allowed")
     zip_code = fields.Char(related="partner_id.zip", store=True)
     city = fields.Char(related="partner_id.city", store=True)
 
+    @api.depends("rule_id", "rule_id.available_to_promise_defer_pull")
+    def _compute_unrelease_allowed(self):
+        user_is_allowed = self.env.user.has_group("stock.group_stock_user")
+        for move in self:
+            unrelease_allowed = (
```

Comment:
@lmignon Shouldn't this kind of rule in a separate method in order allow override?

Reply:
hard to express this kind of check. I prefer to wait for the need of such a split...